README.md

```diff
@@ -30,13 +30,16 @@
 
 
 ## I. Technologies Implemented
-- SendGrid API v3
-- Google Calendar
-
++ SendGrid API v3
```

Comment:
@Briahana-Mauge I had these two separated because they aren't built into our app aka external techs. But the change is fine with me.

Reply:
I put the space back between the two!  Should I add Chartjs to that group as well?

Reply:
Chartjs was imported in so no I don't think so.